services/welding-machine-data-simulator-service/app/services/azure_storage.py

```
@@ -0,0 +1,178 @@
+
+        except Exception as e:
+            print(f"❌ Welding Machine 데이터 시뮬레이션 실패: {e}")
+            return None
+
+    async def _load_dataframes(self):
+        """DataFrame 캐싱을 위한 로드"""
+        try:
+            files = await self.list_data_files()
+
+            if not files:
+                print(f"⚠️ Welding Machine 데이터 파일이 없습니다.")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)